programs/sbf/tests/programs.rs

```diff
@@ -848,33 +1005,49 @@ fn test_program_sbf_invoke_sanity() {
         println!("Test program: {:?}", program);
 
         let GenesisConfigInfo {
-            mut genesis_config,
+            genesis_config,
             mint_keypair,
             ..
         } = create_genesis_config(50);
 
-        // deactivate `disable_bpf_loader_instructions` feature so that the program
-        // can be loaded, finalized and tested.
-        genesis_config
-            .accounts
-            .remove(&feature_set::disable_bpf_loader_instructions::id());
-
-        genesis_config
-            .accounts
-            .remove(&feature_set::deprecate_executable_meta_update_in_bpf_loader::id());
-
         let (bank, bank_forks) = Bank::new_with_bank_forks_for_tests(&genesis_config);
         let mut bank_client = BankClient::new_shared(bank.clone());
 
-        let invoke_program_id =
-            load_program(&bank_client, &bpf_loader::id(), &mint_keypair, program.1);
-        let invoked_program_id =
-            load_program(&bank_client, &bpf_loader::id(), &mint_keypair, program.2);
-        let (bank, noop_program_id) = load_program_and_advance_slot(
+        let authority_keypair = Keypair::new();
+
+        let invoke_buffer_keypair = Keypair::new();
+        let invoke_program_keypair = Keypair::new();
+        let invoke_program_id = invoke_program_keypair.pubkey();
+        load_upgradeable_program(
```

Comment:
Most of the new calls to `load_upgradeable_programs` take in buffer and program keypair as input parameters, but the two values are not used afterwards. We could add a wrapper to the function that generates these two keypairs internally, and just returns the pubkey of program.

Reply:
updated.